lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, subsection, course):
         self.due = getattr(subsection, 'due', None)
         self.graded = getattr(subsection, 'graded', False)
 
-        self.course_version = getattr(course, 'course_version', None)
+        self.course_version = getattr(subsection, 'course_version', None)
```

Comment:
Yes, I have a work-in-progress PR for removing the `course` argument.  However, I'd like to make that change as part of a separate ticket since it's more extensive.  We can do so as part of https://openedx.atlassian.net/browse/TNL-5910 (for use data only from course_structure not course module).